package.json

```diff
@@ -3,8 +3,9 @@
   "version": "3.0.0",
   "main": "app/main.js",
   "scripts": {
-    "start": "http-server -a 0.0.0.0 -p 9000",
-    "bundle": "webpack app/main.js ./main.bundle.js --config webpack.config.js --watch -d"
+    "start": "http-server",
```

Comment:
i think those settings are only for your local branch, do not push this to master